packages/components/src/components/Select/Combobox.tsx

```diff
@@ -91,6 +110,8 @@ export const SelectCombobox = (
         highlightedIndex,
         inputValue,
     } = useCombobox({
+        labelId,
+        id,
```

Comment:
```suggestion
        toggleButtonId: id,
```
Then you can remove the `id={toggleButtonId}` and put back the spread